templates/includes/_main_sponsors.html

```diff
@@ -104,4 +104,14 @@
             <img class="img-responsive center-block" src="{% static 'img/global/supporters/wsp.png' %}" alt="WSP Digital" />
         </a>
     </div>
+    <div class="col-md-3">
```

Comment:
Is there any reason we add extra sponsors here? Are they sponsors who donated through Crowdfunding?

Reply:
Yes, they are new sponsors from crowdfunding campaign as well as new sponsors who came on board after the campaign.